django/db/backends/base/schema.py

```diff
@@ -62,6 +62,8 @@ class BaseDatabaseSchemaEditor:
     sql_alter_column_not_null = "ALTER COLUMN %(column)s SET NOT NULL"
     sql_alter_column_default = "ALTER COLUMN %(column)s SET DEFAULT %(default)s"
     sql_alter_column_no_default = "ALTER COLUMN %(column)s DROP DEFAULT"
+    sql_alter_column_no_default_null = "ALTER COLUMN %(column)s SET DEFAULT NULL"
+
```

Comment:
```suggestion
    sql_alter_column_no_default_null = sql_alter_column_no_default
```

And then override it in `mysql.schema.SchemaEditor.sql_alter_column_no_default_null = "ALTER COLUMN %(column)s SET DEFAULT NULL"`

Reply:
@charettes Okay i understood! Big thanks!